CHANGES.rst

```diff
@@ -51,6 +51,13 @@ Improvements
 - Allow events to be favorited (:issue:`1662`, :pr:`5338`, thanks :user:`Leats`)
 - Include abstract content in CSV/Excel export if enabled in the abstract list
   (:issue:`5356`, :pr:`5372`, thanks :user:`rppt`)
+- Add the ability to include an optional static javascript file when defining
+  custom conference themes from within a plugin.
+  (:pr:`5414`, thanks :user:`brittyazel`)
+- Conference themes are now defined using a new ConferenceTheme dataclass as
+  opposed to using a tuple, though we still maintain backwards compatibility
+  with the old way.
```

Comment:
I would skip this in the changelog tbh. It's really something internal which we typically do not mention there. However, feel free to include a short mention in the "Internal Changes" if you want (personally i'd skip it). If you want to keep it I'd reword it a bit though ("Conference themes from plugins can now be defined using a `ConferenceTheme` object instead of a tuple"). That's more concise and "can" already implies it's backwards compatible.